jesterTOV/inference/flows/models/nicer_maf/J04374715/amsterdam_cst_pdt/submit.sh

```
@@ -6,13 +6,13 @@
 #SBATCH --gpus-per-node=1
 #SBATCH --cpus-per-gpu=1
 #SBATCH --mem-per-gpu=10G
-#SBATCH --output="../../../../../logs/%x.out"
+#SBATCH --output="training_log.out"
 #SBATCH --job-name="amsterdam_cst_pdt"
 
 now=$(date)
 echo "$now"
 echo "Training flow for: amsterdam_cst_pdt"
-source /home/twouters2/projects/jester_review/jester/.venv/bin/activate
+source /home/twouters2/projects/43_eos_bayesian_updates/new_flowjax_jester/.venv/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: /home/twouters2/projects/43_eos_bayesian_updates/new_flowjax_jester/.venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)